charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -60,9 +60,9 @@ spec:
         - --feature-gates
         - OriginatingIdentity=true
         {{- end }}
-        {{- if .Values.namespacedServiceBrokerEnabled }}
+        {{- if not .Values.namespacedServiceBrokerEnabled }}
```

Comment:
How does this helm conditional evaluate if the variable is not defined?  My quick read on this is if the value is set to false or not defined, it will set the NamespacedServiceBroker to false.  I'd like to see this only set it to false if the value is explicitly set to false in values.yaml.  WDYT?